4.x/amd/dojo/build.profile.js

```diff
@@ -60,21 +61,7 @@ var profile = {
       trees: [
         // don"t bother with .hidden, tests, min, src, and templates
         [".", ".", /(\/\.)|(~$)|(node_modules)/],
-      ],
-    },
-    {
-      name: "moment",
-      location: DIR + "/moment",
-      main: "moment",
-      packageJson: {},
-      resourceTags: {
-        miniExclude: function (filename, mid) {
-          if (mid.indexOf("/min/") > -1 || mid.indexOf("/src/") > -1) {
-            return true;
-          }
-          return ["moment/ender", "moment/package"].indexOf(mid) > -1;
-        },
-      },
```

Comment:
Can remove any other packages that the app is not using now that the package has no dependencies.